test/Sentry.Unity.Tests/UnityApplicationLoggingIntegrationTests.cs

```diff
@@ -10,64 +10,67 @@ public sealed class UnityApplicationLoggingIntegrationTests
     {
         private class Fixture
         {
-            public Stubs.TestHub Hub => new();
-            public IApplication Application => new TestApplication();
-
-            public UnityApplicationLoggingIntegration GetSut() => new(Application);
+            public UnityApplicationLoggingIntegration GetSut(IHub hub, SentryOptions sentryOptions)
+            {
+                var application = new TestApplication();
+                var integration = new UnityApplicationLoggingIntegration(application);
```

Comment:
These can be properties. so you can mutate them (to set things up) before calling `GetSut`